src/ray/rpc/grpc_client.h

```diff
@@ -0,0 +1,78 @@
+#ifndef RAY_RPC_GRPC_CLIENT_H
+#define RAY_RPC_GRPC_CLIENT_H
+
+#include <grpcpp/grpcpp.h>
+#include <boost/asio.hpp>
+
+#include "ray/common/grpc_util.h"
+#include "ray/common/status.h"
+#include "ray/rpc/client_call.h"
+
+namespace ray {
+namespace rpc {
+
+// Call a specific rpc method.
+#define RPC_CALL_METHOD(SERVICE, METHOD, request, callback) \
```

Comment:
Does this assume there is a field defined called "rpc_client_"? That seems a little too magical to me, I'd prefer to just have the user always specify the client (just rename RPC_CALL_METHOD_WITH_CLIENT TO RPC_CALL and remove this one)

Reply:
Also please improve the comment to specify what this specifically does and why we have it

Reply:
Makes sense. thanks.